cernopendata/static/templates/search/briefs/dataset.html

```diff
@@ -2,7 +2,7 @@
 <div class="row">
   <div class="col-md-7">
     <h4>
-      <a target="_self" ng-href="/datasets/{{ record.id }}">{{ record.metadata.title }}</a>
+      <a target="_self" ng-href="/datasets/{{ record.metadata.control_number }}">{{ record.metadata.title }}</a>
```

Comment:
Hmm, `control_number` is a post-artefact from the MARC21 world, I think it may be good to replace it... Let's see IRL.

```console
$  git grep control_number
cernopendata/modules/fixtures/cli.py:                assert 'control_number' in record.json
cernopendata/modules/fixtures/cli.py:                ) + str(record.json['control_number'])
cernopendata/modules/records/fetchers/artid.py:        pid_value=data['control_number'],
cernopendata/modules/records/fetchers/recid.py:        pid_value=data['control_number'],
cernopendata/modules/records/fetchers/termid.py:        pid_value=data['control_number'],
cernopendata/modules/records/minters/artid.py:    data['control_number'] = provider.pid.pid_value
cernopendata/modules/records/minters/recid.py:    data['control_number'] = provider.pid.pid_value
cernopendata/modules/records/minters/termid.py:    data['control_number'] = provider.pid.pid_value
cernopendata/static/templates/cernopendata_terms_ui/brief.html:      <a target="_self" ng-href="/glossary/{{record.metadata.control_number}}">
cernopendata/static/templates/faceted_results.html:          <h3><a ng-href="/articles/{{record.metadata.control_number}}">{{record.metadata.title}}<a/></h3>
cernopendata/static/templates/resources/results.html:          <h3><a ng-href="/articles/{{record.metadata.control_number}}">{{record.metadata.title}}<a/></h3>
cernopendata/static/templates/search/briefs/articles.html:  <a target="_self" ng-href="/articles/{{record.metadata.control_number}}">
cernopendata/static/templates/search/briefs/terms.html:    <a target="_self" ng-href="/glossary/{{record.metadata.control_number}}">
cernopendata/templates/cernopendata_pages/index.html:                  <h3><a href="/articles/{{r['_source']['control_number']}}">{{r['_source']['title']}}</a></h3>
```

Reply:
I updated the PR and fixed the comments with proposed changes. Regarding this one I left it for now with control_number as we are not sure how we are going to proceed.

Reply:
@tiborsimko a solution would be to do directly ```record.metadata.doi``` and skip the control number. What do you think ?